ddtrace/propagation/http.py

```diff
@@ -71,20 +79,18 @@ def _extract_header_value(possible_header_names, headers, default=None):
     return default
 
 
-def _b3_id_to_dd_id(b3_id):
+def _hex_id_to_dd_id(hex_id):
```

Comment:
A bit of a nit...: not sure it actually makes sense to rename this function. Just because B3 happens to use the same "hex" spec that OTel does, doesn't mean that they will remain the same in the future. By merging them into the same function we might implicitly break support for one when changing the function for the other (eg. if we need to add something for OTel we might break B3). 

My preference would be that we keep the `_b3_id_to_dd_id` and just set it equal to `_hex_id_to_dd_id` for now since they happen to be the same. If the implementations were to diverge then `_b3_id_to_dd_id`/`_otel_id_to_dd_id` could be modified to handle the specific cases.

This is probably a bit overkill at the moment, but maybe something to think about now.